source/_integrations/smarty.markdown

```diff
@@ -5,6 +5,7 @@ ha_category:
   - Hub
   - Fan
   - Sensor
+ha_iot_class: Local Push
```

Comment:
It is a centralized polling integration. In the main module of this integration, a 30 second timer is used to centrally poll the API and next notify the entities of the update.

This makes this integration a polling integration.

```suggestion
ha_iot_class: Local Polling
```

Reply:
Clear, thanks.